src/panel.css

```diff
@@ -0,0 +1,41 @@
+.wrapper {
+    width: 280px;
+    height: 170px;
+    padding:10px 10px 10px 10px;
+    box-shadow: 0 8px 40px rgba(#000000, .2);
+}
+      
+.wrapper .profile {
+          line-height: 1.5em;
+          margin-top: 0.5em;
+          position: relative;
+          border-bottom:1px solid #dbd5d5;
+          zoom: 1;
+          padding-bottom:10px;
+}
+          
+.profile :after {
+              width: 100%;
```

Comment:
Check the alignment in this file as it's not consistent with itself. I suspect we should use 2 spaces like is most common in Mozilla code.